src/ts2gas.cjs

```diff
@@ -0,0 +1,5 @@
+if (require.main === module) {
+  import('./index.js').then(index => {
+    process.stdout.write(index.default(process.argv[2]));
+  });
+}
```

Comment:
Why not simply use an older version of this package?

Reply:
https://github.com/grant/ts2gas/pull/64/files#diff-b55cdbef4907b7045f32cc5360d48d262cca5f94062e353089f189f4460039e0L20-R20

I would rather use ES2019 as my target, and new targets going forward.

Reply:
Perhaps a  better approach would be to:

- Update Typescript dependency, thus benefiting from latest TS features
- Change library build process to produce and offer  `.cjs`  and `.mjs`

Reply:
I've been spending more time in the development environment, and have found a lot of shortcomings with this approach and believe I have found a better solution, especially after careful study of the starter project.

The short summary is to generate an incremental `src/` project as a "normal" module-based typescript project, and reference that with another "normal" typescript project in `test/`.

The key is to create a third typescript project in `dist/`, which uses a `predist` npm script step to copy over the files from src, (truncating the last line, which *must always contain* the `export` statement 😉 ) before applying the all-in one file that gets pushed by clasp.

This is great since I can import my namespace-declared modules for tests, and get proper type hints, etc. when developing, without imposing much in the way of process or ceremony in the project source code.